hackathon_site/hardware/fixtures/hardware.yaml

```diff
@@ -0,0 +1,146 @@
+-   model: hardware.category
```

Comment:
Nice. Can you add something to the readme about how to load this, and whether or not it is loaded during testing?

Reply:
Reminder of this, will be helpful for the people less experienced with django